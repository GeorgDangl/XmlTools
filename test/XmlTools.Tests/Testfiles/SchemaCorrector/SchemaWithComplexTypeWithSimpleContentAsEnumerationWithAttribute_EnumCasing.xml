<?xml version="1.0" encoding="UTF-8"?>
<Monsters>
  <Monster Resistance="Fire">DRAGON</Monster>
  <Monster Resistance="Poison">ORC</Monster>
  <Monster Resistance="ICE">Ogre</Monster>
  <Monster Resistance="ice">Lich</Monster>
</Monsters>

