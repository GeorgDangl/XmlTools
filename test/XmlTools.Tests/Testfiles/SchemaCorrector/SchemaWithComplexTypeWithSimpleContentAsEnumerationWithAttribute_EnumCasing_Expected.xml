<?xml version="1.0" encoding="UTF-8"?>
<Monsters>
  <Monster Resistance="Fire">Dragon</Monster>
  <Monster Resistance="Poison">Orc</Monster>
  <Monster Resistance="Ice">Ogre</Monster>
  <Monster Resistance="Ice">Lich</Monster>
</Monsters>

